core/metacling/src/TClingCallFunc.cxx

```diff
@@ -479,6 +510,9 @@ const clang::DeclContext *TClingCallFunc::GetDeclContext() const {
 
 int TClingCallFunc::get_wrapper_code(std::string &wrapper_name, std::string &wrapper)
 {
+   // Could trigger deserialization of decls.
+   cling::Interpreter::PushTransactionRAII RAII(fInterp);
```

Comment:
We should probably annotate the copy ctor iteration with these raiis to be precise on where deserialization happens.

Reply:
Ok, annoying, indeed. Maybe in the outer argument loop and have a comment that we eventually iterate over ctors which deserializes is going to be enough.

Reply:
Ok, I'm going to do that.